plugins/synaptics-rmi/fu-synaptics-rmi-v5-device.c

```diff
@@ -8,8 +8,14 @@
 
 #include "config.h"
 
+#ifdef HAVE_GNUTLS
```

Comment:
since it's a hard requirement now for the plugin shouldn't all the `#ifdef HAVE_GNUTLS` go away?

Reply:
Agree; done.